### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by MayPyone**
-Updated and created tables
-Inserted data
-Use
Screenshot (653)
Screenshot (655)
Screenshot (656)

Screenshot (654)
 'join' for queries

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -56,4 +56,58 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
...
+    /* number of animals of per species */
+      select species.name, count(animals.id)
+      from animals
+      join species
+      on animals.species_id = species.id group by species.name;
+    /* owner and animal */
+      select owners.full_name, animals.name
+      from animals
+      join owners
+      on animals.owner_id = owners.id;
```
Comment (shahadat3669) [inline queries.sql]: - [ ] Your query is running, but there's a potential issue. Currently, you're querying to get animals per owners, but there could be a scenario where someone like `Jodie Whittaker` doesn't have any animals. As per the project requirement, you should "List all owners and their animals, remember to include those that don't own any animal." This means your query should also display owners who don't have any animals. Please update your code to fulfill this requirement, ensuring that all owners are included in the results, whether they have animals or not. Keep up the good work! 🐾👥🦔

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @MayPyone , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description
✅ The code is top-notch! 💯


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MayPyone** [comment]
Screenshot (657)

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @MayPyone , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implemnting the requested change. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**